webapp/components/NotificationsTable/NotificationsTable.vue

```diff
@@ -23,7 +28,7 @@
         </client-only>
       </ds-space>
       <ds-text :class="{ 'notification-status': scope.row.read, reason: true }">
-        {{ $t(`notifications.reason.${scope.row.reason}`) }}
+        {{ notificationReason(scope.row) }}
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-13T08:58:53Z" title="Thursday, February 13th 2020, 9:58:53 am +01:00">Feb 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,7 +28,7 @@
         </client-only>
       </ds-space>
       <ds-text :class="{ 'notification-status': scope.row.read, reason: true }">
-        {{ $t(`notifications.reason.${scope.row.reason}`) }}
+        {{ notificationReason(scope.row) }}
```


The decision details which you skipped to implement like I explained above have to be shown in this file @roschaefer !!!